internal/extproc/translator/translator.go

```diff
@@ -65,7 +71,7 @@ type Translator interface {
 	ResponseBody(body io.Reader, endOfStream bool) (
 		headerMutation *extprocv3.HeaderMutation,
 		bodyMutation *extprocv3.BodyMutation,
-		usedToken uint32,
+		tokenUsage *TokenUsage,
```

Comment:
i don't think this three uint64 would be worth the allocations, so could you return the value
```suggestion
		tokenUsage ToenUsage,
```

Reply:
kept it as a pointer so that `nil` value would indicate absence of tokenUsage data in the responseBody
`if tokenUsage == nil {...}`
with struct will have to check zero value for one of the member
`if tokenUsage.TotalTokens == 0 {...}`

thoughts?